draft-ietf-quic-transport.md

```diff
@@ -4254,39 +4251,37 @@ stream data that it has sent for the affected stream, unless this is a result of
 a change in the initial limits (see {{zerortt-parameters}}).
 
 
-## MAX_STREAM_ID Frame {#frame-max-stream-id}
+## MAX_STREAMS Frame {#frame-max-streams}
 
-The MAX_STREAM_ID frame (type=0x06) informs the peer of the maximum stream ID
-that they are permitted to open.
+The MAX_STREAMS frame (type=0x1c and 0x1d) informs the peer of the number of
+streams it is permitted to open.  A MAX_STREAMS frame with a type of 0x1c
+applies to bidirectional streams; a MAX_STREAMS frame with a type of 0x1d
+applies to unidirectional streams.
 
 The frame is as follows:
 
 ~~~
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                    Maximum Stream ID (i)                    ...
+|                     Maximum Streams (i)                     ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ~~~
 
-The fields in the MAX_STREAM_ID frame are as follows:
+The fields in the MAX_STREAMS frame are as follows:
 
-Maximum Stream ID:
-: ID of the maximum unidirectional or bidirectional peer-initiated stream ID for
-  the connection encoded as a variable-length integer. The limit applies to
-  unidirectional steams if the second least signification bit of the stream ID
-  is 1, and applies to bidirectional streams if it is 0.
+Maximum Streams:
+: A count of the number of streams that can be opened.
```

Comment:
```suggestion
: A count of the total number of streams of a given type that can be opened over the course of the connection.
```

This is potentially ambiguous, because it could be read as "100 = 100 open streams at once."